### DISCUSSION TITLE
How do you manage your Doctrine Migrations ?

### DISCUSSION THREAD

**Discussion by YannickDurden** [symfony/symfony#50903] [Q&A]
Hi mates,

I'm currently seaching a solution to keep the Doctrine Migrations clean and easy to manage across the time for a large project.

Actually, I'm using Migration "classically": 

1/ After entity modifications, I'm using the CLI to generate a new migration
2/ I run the new migration

or

1/ I generate a new and empty migration and I add my own SQL modification
2/ I run the migration

If we have many releases, you cannot know that if you open the Doctrine Migration directory.

How do you manage your migrations? Do you edit the file name? Are you using subdirectories to handle many releases?

Thanks mates for your feedback :zap:

**Comment by tourze** [comment]
I used to manage the table structure by making migration file.
After two years, I have about two hundred Entity file, and  over two thousand migration files...
For saving my life, I delete all migration files, and only use two simple command:

```
bin/console doctrine:schema:update --dump-sql # Diff the table structure and entity definition, print all sql.
bin/console doctrine:schema:update --dump-sql -f # Run all diff sql
```

If there are any SQL statements that I am not sure will have an impact on production, I will send all sql to another DBA to check and run it.
